Cesium.CodeGen/Contexts/AssemblyContext.cs

```diff
@@ -37,6 +37,7 @@ public class AssemblyContext
 
         var targetRuntime = compilationOptions.TargetRuntime;
         assembly.CustomAttributes.Add(targetRuntime.GetTargetFrameworkAttribute(module));
+        module.AssemblyReferences.Add(targetRuntime.GetSystemAssemblyReference());
```

Comment:
_(Minor)_ This line was removed in #385, but here it's back. I assume we should actually drop that?

The new tests work with it, though, so I assume it doesn't matter, and I merge :)

Reply:
My general policy is to make all changes independent. and this change really was not important for global statics, so I only reduce changeset. It was a slip on my part to put removal in PR.

Reply:
Actually, it seems kinda a rebase/merge artifact rather than an actual change.